ISISComputingGroup/EPICS-ioc
# Ticket980 982 danfysik io cs

ISISComputingGroup/IBEX#980 and ISISComputingGroup/IBEX#982

To Test:
Re-make your whole instrument
Asking for a console with -x qualifier you should now see DFKPS01 through to 09
Run any number of the IOCs you like, all should have records relating to things such as FIELD
Putting it into SIM should allow for the values through the current to field, and vice versa to update as appropriate
Changing the field unit should also alter the field value

Note without a Danfysik to test with, this can only be simulated - I shall create some tickets for the final verification with an actual danfysik
